fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -234,7 +235,7 @@ private boolean syncAndCheckPartitions(TaskRunContext context,
                                            Map<TableSnapshotInfo, Set<String>> baseTableCandidatePartitions)
             throws AnalysisException, LockTimeoutException {
         // collect partition infos of ref base tables
-        LOG.info("start to sync and check partitions for mv: {}", materializedView.getName());
+        LOG.debug("Start to sync and check partitions for mv: {}", materializedView.getName());
```

Comment:
is there probability that debug log is overwhelming ?

Reply:
Yes, and it contains no useful info.